src/status_emitter.rs

```diff
@@ -243,16 +243,14 @@ impl TestStatus for TextTest {
         stderr: &'a [u8],
         stdout: &'a [u8],
     ) -> Box<dyn Debug + 'a> {
+        let mut path = self.path.display().to_string();
+        if !self.revision.is_empty() {
+            write!(path, " (revision `{}`)", self.revision).unwrap();
+        }
```

Comment:
Isn't there a function that returns this name-plus-revision? It's also printed while the tests are running, right? We should have that logic only once to keep it consistent...

Reply:
Ah, found that function. It wasn't consistent, so the output changed a bit.